src/visualization/types/Graph/view.tsx

```diff
@@ -14,6 +14,22 @@ import {
 // Components
 import EmptyGraphMessage from 'src/shared/components/EmptyGraphMessage'
 
+// Context
+import {AppSettingContext} from 'src/shared/contexts/app'
+
+// Redux
+import {writeThenFetchAndSetAnnotations} from 'src/annotations/actions/thunks'
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+// Constants
+import {VIS_THEME, VIS_THEME_LIGHT} from 'src/shared/constants'
+import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
+import {INVALID_DATA_COPY} from 'src/visualization/constants'
+
+// Types
+import {AppState, XYViewProperties} from 'src/types'
+import {VisualizationProps} from 'src/visualization'
+
```

Comment:
dumb diffing tool. i moved the `// Utils` section under this section and it thinks i moved this section up.